src/com.company/backtracking/PalindromePartitioning.java

```diff
@@ -0,0 +1,38 @@
+package backtracking;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class PalindromePartitioning {
+
+    void palindromeParitioning(String string, int index, List<String> currentList, List<List<String>> result) {
+        if(index >= string.length()) {
+            result.add(new ArrayList<>(currentList));
+            return;
+        }
+
+        for(int i = index; i< string.length(); i++) {
+            if(isPalindrome(string.substring(index, i + 1))) {
+                currentList.add(string.substring(index, i + 1));
+                palindromeParitioning(string, index + 1, currentList, result);
+                currentList.remove(currentList.size() - 1);
+            }
+        }
+    }
+
+    boolean isPalindrome(String string) {
```

Comment:
Add access specifier for all the methods.

Reply:
added for all the methods